### PR TITLE
Add `loading()` method to `Img` tag

### PR DESCRIPTION
**Pull request by jacobbudin**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | #138

### CODE REVIEW

**Review by vjik — CHANGES_REQUESTED** [review]
Looks good 👍
Can you add line to changelog?

**Comment by jacobbudin** [comment]
> Can you add line to changelog?

@vjik Yes. Done.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## 2.5.1 under development
 
+- Chg #139: Add `loading()` method to `Img` tag (@jacobbudin)
```
Comment (vjik) [inline CHANGELOG.md:5]:
```suggestion
- New #139: Add `loading()` method to `Img` tag (@jacobbudin)
```

**Review by vjik — APPROVED** [review]

**Comment by vjik** [comment]
Thank you 👍

**Pull request merged**